apps/civicsignalblog/.eslintrc.js

```diff
@@ -6,6 +6,16 @@ module.exports = {
       webpack: {
         config: "./eslint.webpack.config.js",
       },
+      "eslint-import-resolver-custom-alias": {
+        alias: {
+          "#civicsignalblog": "./src",
+        },
+        extensions: [".js"],
+      },
```

Comment:
`eslint` should pick these up if defined in [`jsconfig`](https://github.com/CodeForAfrica/ui/blob/30aea2de7617e9270f212ddcbf88543eface4e51/packages/eslint-config-commons-ui/index.js#L56-L61), no?

Reply:
Makes sense, should work 🚀